src/remotebuild/lib/server.ts

```diff
@@ -51,6 +53,13 @@ class Server {
         var app = express();
         app.use(expressLogger("dev"));
         app.use(errorhandler());
+        if (conf.secure) {
+            app.use(helmet.hsts({
+                maxAge: 1000 * 60 * 60 * 24 * 365, // one year in milliseconds
+                includeSubdomains: false,
```

Comment:
Can you please add comments explaining why did we choose to set this property to false? (Same for all other properties)